helm/docs/expose-teams-api.md

```diff
@@ -11,23 +11,41 @@
 
 # Exposing the `teams-api` Service
 
-You may wish to expose your FiftyOne Teams API for SDK access.
+There are two methods for SDK access to Fiftyone Teams
 
-You may expose your `teams-api` service in any manner that suits your deployment strategy.
-The following is one solution, but does not represent the entirety of possible solutions.
-Any solution allowing the FiftyOne Teams SDK to use websockets to access the `teams-api` service on port 80 should work.
+- Direct MongoDB connection
+- FiftyOne Teams API
 
-**NOTE**: The `teams-api` service uses websockets to maintain connections and allow for long-running processes to complete.
-Please ensure your Infrastructure supports websockets before attempting to expose the `teams-api` service.
-(e.g. You will have to migrate from AWS Classic Load Balancers to AWS Application Load Balancers to provide websockets support.)
+The database direct connection requires each user to have root database privileges.
+This is not a preferred security posture and presents risks.
```

Comment:
This feels unnecessary - just leave it at the fact that db-direct access is root database permissions.

Reply:
I'll remove it. I added it to call out why one may prefer to use the API.